### PR TITLE
[CAS-1070] Restructure search view component docs

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://stream-io.atlassian.net/browse/CAS-1070

### 🎯 Goal

Restructure search view component docs

### 🛠 Implementation details

Well, there are no ways to customize these views so skipping the corresponding section.

### 🎨 UI Changes

screencapture-localhost-3000-chat-docs-sdk-android-ui-components-search-view-2021-06-24-03_11_46

### 🧪 Testing

- Run docs locally `npx stream-chat-docusaurus -i -s`
- Open Search View component section and read it.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### 🎉 GIF

giphy (4)

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/04-components/07-search-view.md
Comment (adasiewiczr) [inline docusaurus/docs/Android/03-ui/04-components/07-search-view.md:90]:
```suggestion
The full list of listeners available for `SearchResultListView` can be found [here](https://getstream.github.io/stream-chat-android/stream-chat-android-ui-components/stream-chat-android-ui-components/io.getstream.chat.android.ui.search.list/-search-result-list-view/index.html).
```

**Review by adasiewiczr — APPROVED** [review]
LGTM but let's wait for a second review

File: docusaurus/docs/Android/03-ui/04-components/07-search-view.md
Comment (bychkovdmitry) [reply docusaurus/docs/Android/03-ui/04-components/07-search-view.md:90]: Thanks 🤦

**Pull request merged**
